examples/package.json

```diff
@@ -7,24 +7,25 @@
     "build:client": "cd .. && yarn build",
     "_print_starting_label": "echo '' && echo '========== Starting ==========' && echo ''",
     "run:example": "yarn build:client && yarn _print_starting_label && node --loader ts-node/esm --no-warnings=ExperimentalWarning",
-    "access-key": "yarn run:example access-key.ts",
-    "admin-add-user": "yarn run:example admin-add-user.ts",
     "admin-webhooks": "yarn run:example admin-webhooks.ts",
+    "admin-add-user": "yarn run:example admin-add-user.ts",
     "admin-remove-user": "yarn run:example admin-remove-user.ts",
-    "create-transaction": "yarn run:example create-transaction.ts",
-    "custom-logger": "yarn run:example custom-logger.ts",
-    "generate-keys": "yarn run:example generate-keys.ts",
-    "get-balance": "yarn run:example get-balance.ts",
-    "get-handled-paymail-domains": "yarn run:example get-handled-paymail-domains.ts",
-    "get-shared-config": "yarn run:example get-shared-config.ts",
-    "handle-exceptions": "yarn run:example handle-exceptions.ts",
-    "list-transactions": "yarn run:example list-transactions.ts",
-    "manage-contacts": "yarn run:example manage-contacts.ts",
-    "send-op-return": "yarn run:example send-op-return.ts",
-    "sync-merkleroots": "yarn run:example sync-merkleroots.ts",
-    "update-user-xpub-metadata": "yarn run:example update-user-xpub-metadata.ts",
+    "admin-get-handled-paymail-domains": "yarn run:example admin-get-handled-paymail-domains.ts",
+    "user-generate-totp": "yarn run:example generate-totp.ts",
```

Comment:
This line is unnecessary, we should use:
`"manage-contacts": "yarn run:example manage-contacts.ts",` instead.